wiki.py

```diff
@@ -308,6 +366,26 @@ def _make_year_title(self, raw_string):
         title = raw_string.replace("<YEAR>", str(self._year))
         return title
 
+    def _prompt_year_page(self, title):
+        """Prompt user for writing year page.
+
+        Parameters
+        ----------
+        title : str
+            Title of the year page.
+
+        Returns
+        -------
+        bool
+            True if the prompt year pages command line is set or if
+            the user answers "y", else False.
+        """
+        if not self._prompt_year_pages:
+            return True
+
+        print('Add page "{}"? (y/N)'.format(self._make_year_title(title)))
```

Comment:
why not use `input(...)` or `raw_input(...).lower()`?

Reply:
Added prompt to input.